client/src/main/java/com/flipkart/flux/client/runtime/FluxHttpClient.java

```diff
@@ -0,0 +1,6 @@
+package com.flipkart.flux.client.runtime;
+
+public interface FluxHttpClient {
+
+    void postOverHttp(Object dataToPost, String pathSuffix);
```

Comment:
while returning void works for our current use cases, will face issues when we get the response after doing post. This same API is used by both Client and Task runtime to connect to Flux runtime.